### PR TITLE
initial commit of ampvis2

### PR DESCRIPTION
**Pull request by bernt-matthias**
This adds the ampvis2 tools   https://madsalbertsen.github.io/ampvis2/articles/ampvis2.html for visualizing amplicon data (and probably more .. essentially abundance + metadata + taxonomy). 

The `load` tool creates an 

1. `rds` dataset
2. a restructured metadata table 
3. a list of taxons dataset 

that are used by the other tools. 2 + 3 are only used by the other tools to create useful select parameters (essentially because the dataset metadata of the Galaxy biom datatypes do not allow for this ... )

TODOs

- [x] tools use `rds` as datatype -> define `ampvis_rds` datatype for this tool  https://github.com/galaxyproject/galaxy/pull/13083 / https://github.com/galaxyproject/galaxy/pull/13084
- [x] the load tool needs https://github.com/galaxyproject/galaxy/pull/13048 and a backport 
- [x] add export_fasta?
- [x] specify defaults for options not via tokens, but with duplicated options

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by bgruening — APPROVED** [review]
My only question is if we can have one tool and a select for plotting to avoid passing rds around.

**Review by bgruening — APPROVED** [review]
LGTM, my only question is if we can have just one tool with a select of the various visualisations. This would avoid passing around RDS.

**Comment by bernt-matthias** [comment]
> LGTM, my only question is if we can have just one tool with a select of the various visualisations.

I'm a bit afraid that this single tool might become too complex/large. 

Also the metadata preprocessing that allows for a convenient selection of metadata columns / values in downstream tools seems difficult/impossible to realize in a single tool (one would need a column selector and another select that allow to choose from this column .. and I have no idea how to realize this). 

> This would avoid passing around RDS.

Still did not understand why this is a problem :)

File: tools/ampvis2/export_fasta.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<tool id="ampvis2_export_fasta" name="ampvis2: export fasta" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/ampvis2/export_fasta.xml]: Good point. Fixed.

**Review by bgruening — APPROVED** [review]
Do not forget to manually push to the TS until the lint error is not fixed.

**Comment by bernt-matthias** [comment]
Seems that I can not merge.

**Pull request merged**